docs/macOS-install-and-tutorial-instructions-for-conda.md

```diff
@@ -0,0 +1,35 @@
+# Installing brainlit on macOS using conda environment:
+1. Installation of conda environment
+    * [get Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html)
+    * create a virtual environment: `conda create --name brainlit python=3.8`
+    * activate the environment: `conda activate brainlit`
+2. Install brainlit
+    * Option 1: install from pypi
+        * install brainlit: `pip install brainlit`
+    * Option 2: install brainlit from source
+        * cd to directory you want to install brainlit in
+        * clone the repo: git clone https://github.com/neurodata/brainlit.git
+        * cd into the repo: `cd brainlit`
+        * install brainlit: `pip install -e .`
+
+# Downloading brains tutorial on iOS using conda environment, using jupyter notebook: 
+* in a new terminal: `conda activate brainlit`
+* cd to directory brainlit is installed
+* `cd brainlit/docs/notebooks/utils`
+* run jupyter notebook: `jupyter notebook`
+* select `downloading_brains.ipynb`
+* run tutorial
+
+# Common issues and fixes:
```

Comment:
Seems like this is "Common issues and fixes with downloading_brains notebook?" right? If so, please specify. If not, I'm not sure what you mean by "Section (2)" below

Reply:
Specified section title as "Common issues and fixes with downloading_brains notebook" and changed bullet point to "Section (2) of downloading_brains notebook" and "Section (4) of downloading_brains notebook"